Debian/bin/lib/Cyrus/Docker.pm

```diff
@@ -10,7 +11,15 @@ use App::Cmd::Setup 0.336 -app => {
 sub repo_root ($self) {
   $self->{root} //= do {
     my $path = $ENV{CYRUS_CLONE_ROOT} || '/srv/cyrus-imapd';
-    path($path);
+    Path::Tiny::path($path);
+  };
+}
+
+sub config ($self) {
+  $self->{config} //= do {
+    my $path = Path::Tiny::path('/etc/cyrus-docker.json');
+    $path ? decode_json($path->slurp)
```

Comment:
`$path` will always be true so this will try to slurp a file that doesn't exist and crash. I think you want `$path->exists` or `$path->is_file`

Reply:
With `use JSON::XS ()` `decode_json` isn't imported so this crashes.

Reply:
fixed in fixup